prow/cluster/monitoring/README.md

```diff
@@ -62,15 +62,15 @@ Developing a new dashboard can be achieved by
 
     ```
     ### if you want to take a look at some json file, eg, hook.json
-    $ bazel build //prow/monitoring/mixins/grafana_dashboards:hook
-    $ cat bazel-bin/prow/monitoring/mixins/grafana_dashboards/hook.json
+    $ bazel build //prow/cluster/monitoring/mixins/grafana_dashboards:hook
```

Comment:
Can we split these out into a separate change?

Reply:
Done.